packages/wdio-cli/src/utils.js

```diff
@@ -213,3 +214,32 @@ export async function missingConfigurationPrompt(command, message, useYarn = fal
 
     return await runConfig(useYarn, true)
 }
+
+export function getCapabilities(arg) {
+    const capabilities = getPlatformCaps(arg.option)
+    let finalCapabilities = capabilities
+    if (capabilities.platformName === IOS_CONFIG.platformName) {
+        finalCapabilities = Object.assign( {},
+            capabilities,
+            {
+                deviceName: arg.device || IOS_CONFIG.deviceName,
+                platformVersion: arg.ver || null,
+                udid: arg.udid || null
```

Comment:
1. Can the repl run with these 2 values as null?
2. What if the passed capabilities are for running a browser and not an app? Do we still need to attach these app capabilities to the repl?

Reply:
1. Yes, it runs as usual.
2. Yes, we still need them to point to correct simulator/real device.